components/cli/pkg/util/utils.go

```diff
@@ -490,6 +493,18 @@ public function main(string action, cellery:ImageName iName, map<cellery:ImageNa
 	return targetFilePath, nil
 }
 
+// returns if the given file or directory exists
+func IsExists(filepath string) (bool, error) {
```

Comment:
suggestion to rename to FileExists(filepath string)

Reply:
@azinneera please note.